tests/wikidata/test_queries.py

```diff
@@ -26,3 +27,9 @@ def test_bind_sparql_query():
 
     with pytest.raises(Exception):
         bind_sparql_query("{{x}}", x='1', y="2")
+
+    with pytest.raises(NotImplementedError):
+        bind_sparql_query("{{x}}", x=[1, 2, 3])
+
+    with pytest.raises(NotImplementedError):
+        bind_sparql_query("{{x}}", x=["a", "b", 3])
```

Comment:
could be beneficial to have additional context for the tests in the shape of comments. Or if pytest implements some kind of labels, then those could be used.

Now it is a bit ambiguous on what is unimplemented

Reply:
Thanks! Refactored the tests into separate functions to and the code to distinct possible, but not implemented, inputs and erroneous inputs.